engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/semantic/RuntimeManagementTest.scala

```diff
@@ -5,11 +5,20 @@ import org.enso.interpreter.test.{InterpreterContext, InterpreterTest}
 import org.enso.common.{LanguageInfo, MethodNames}
 
 import scala.ref.WeakReference
-import scala.util.Try
+import org.graalvm.polyglot.Context
+import org.enso.common.RuntimeOptions
 
 class RuntimeManagementTest extends InterpreterTest {
+  private def parallelism = 5
+
   override def subject: String = "Enso Code Execution"
 
+  override def contextModifiers: Option[Context#Builder => Context#Builder] =
+    Some(b => {
+      b.allowCreateThread(true)
+        .option(RuntimeOptions.JOB_PARALLELISM, "" + parallelism)
```

Comment:
- this test no longer allocates its own `new Thread`, but it relies on builtin `JOB_PARALLELISM`
- this could help a lot with - #10525
- enough to increase value of `JOB_PARALLELISM` (default remains `1`)
- schedule _visualization computations_ as `getThreadManager.submit`
- and they will be executed in parallel to regular computation